classes/rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+
+    @book = book
+
+    @person = person
+  end
```

Comment:
- [ ] Please make sure to achieve the 'belongs-to' association by doing the following:

```ruby
    @book = book
    book.rentals << self

    @person = person
    person.rentals << self
```